extensions/amp-lightbox-gallery/1.0/base-element.js

```diff
@@ -82,7 +82,7 @@ export class BaseElement extends PreactBaseElement {
     toggleAttribute(this.element, 'open', false);
     toggle(this.element, false);
 
-    this.removeAsContainer();
```

Comment:
Are we planning on shipping a separate binary for bento mode? If yes, then in the future we could add a new compile-time constant to let us DCE all the AMP-only code.